### ISSUE TITLE
Issues about -list

### ISSUE THREAD

**Issue by tjhwangxiong** [bioinfo-biols/CIRI-vis#1]
Dear Sir,

If I have 10 samples (5 vs 5), should I combine all the circRNA ID into list.txt?

**Comment by Kevinzjy** [comment]
Yes, you can combine all circRNA ids or just use the ones you want to plot.
